polls/forms.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+from django import forms
+from polls.models import UserProfile
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+class UserCreateForm(forms.Form):
+    BIRTH_YEAR_CHOICES = []
+    for x in range(1900, int(datetime.now().year) + 1):
+        BIRTH_YEAR_CHOICES.append(x)
+
+    SEX = (('M', _('Male')), ('F', _('Female')))
+    username = forms.CharField(label=_('Username'), min_length=4, max_length=150, widget=forms.TextInput(
```

Comment:
ogólnie, żeby nie pisać tych klas w modelu formularza, a żeby wszystko związane z wyglądem było w templatce, to można używać
https://github.com/jazzband/django-widget-tweaks